src/algorithms/Algorithm_Brute_Force.py

```diff
@@ -0,0 +1,123 @@
+import random
+
+
+class AlgorithmBruteForce(object):
+
+    def __init__(self):
+        """
+        EMPTY_SUDOKU -- empty grid with dimension 9x9.
+        DIMENSION -- grid dimension expected, to load the grid from a string.
```

Comment:
There are not parameters in your constructor. You should not comment the global variables in the method comment , you could add single comments next to the variables.